src/components/forms/OperationForm.vue

```diff
@@ -109,12 +110,21 @@ export default defineComponent({
     },
     amount (): number {
       return this.entireCalcul(this.amountString)
+    },
+    account (): Account | null {
+      for (const account of this.$store.state.accounts) {
```

Comment:
it would be useful to transfer this function to the StoreHandler who already have a getCategoryById function. A getAccountById would be useful and could be called here and anywhere else in the future